### ISSUE TITLE
Galaxy run configuration does not respect working directory

### ISSUE THREAD

**Issue by mdklatt** [mdklatt/idea-ansible-plugin#1]
Running from PyCharm with the local project directory set as the working directory and all Ansible files in the `ansible` directory:

```
sh -c ".venv/bin/ansible-galaxy collection install -r ansible/requirements.yml -p ansible/collections && .venv/bin/ansible-galaxy role install -r ansible/requirements.yml -p ansible/roles"
/bin/sh: .venv/bin/ansible-galaxy: No such file or directory

Process finished with exit code 127
```

Activating the virtualenv environment or modifying `$PATH` does not help.

**Comment by mdklatt** [comment]
Cannot reproduce with existing unit test, but code inspection reveals a problem:

```kotlin
        val compoundCommand = sequenceOf(
            getInstallCommand("collection", collectionsDir),
            getInstallCommand("role", rolesDir),
        ).map { it.commandLineString }.joinToString(" && ")
        return PosixCommandLine("sh", "-c", compoundCommand)
```

If a custom collections and/or roles directory is used, separate `ansible-galaxy` calls are required. A single command line is necessary for this, *i.e.* `sh -c '<collections command> && <install command>`. The `PosixCommandLine` instance for this is generated from the string values of the individual install commands. However, the working directory of the combined command is never set to match the individual commands.

The short-term fix is to set the working directory on the combined `PosixCommandLine` instance. Ideally, `PosixCommandLine` in [idea-common](https://github.com/mdklatt/idea-common) should support invoking multiple commands as a single transaction.

**Comment by mdklatt** [comment]
Issue resolved by commit 31db747c1e81375dcec3607812d26119ee56ad7b.
